packages/indexer/src/plugins/kv.ts

```diff
@@ -1,128 +1,137 @@
 import assert from "node:assert";
 import type { Cursor, DataFinality } from "@apibara/protocol";
-import { type Database, type ISqlite, open } from "sqlite";
+import Database, { type Database as SqliteDatabase } from "better-sqlite3";
 import { useIndexerContext } from "../context";
 import { deserialize, serialize } from "../vcr";
 import { defineIndexerPlugin } from "./config";
 
-type SqliteArgs = ISqlite.Config;
+type SqliteArgs = Database.Options & {
```

Comment:
```suggestion
type SqliteArgs = { database: SqliteDatabase }
```

Not sure that's the correct type, but hope you get the idea!